rotkehlchen/db/filtering.py

```diff
@@ -1000,12 +1001,19 @@ def make(
                 column='identifier',
                 values=identifiers,
             ))
-        if ignored_assets_identifiers is not None:
+        if ignored_assets_filter == 'only_ignored':
```

Comment:
you can probably make the code shorter and more readable here and avoid extra if checks and type ignores

`ignored_assets_operator` is the operator you will pass directly in the function. Either `IN` or `NOT IN`

```python
if ignored_assets_identifiers is not None:
            filters.append(DBMultiStringFilter(
                and_op=True,
                column='identifier',
                operator=ignored_assets_operator,
                values=ignored_assets_identifiers,
            ))